rest/route/project_test.go

```diff
@@ -776,3 +776,45 @@ func TestDeleteProject(t *testing.T) {
 	resp = pdh.Run(ctx)
 	assert.Equal(t, http.StatusBadRequest, resp.Status())
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// Tests for PUT /rest/v2/projects/variables/rotate
+
+type ProjectPutRotateSuite struct {
+	sc *data.MockConnector
+	rm gimlet.RouteHandler
+
+	suite.Suite
+}
+
+func TestProjectPutRotateSuite(t *testing.T) {
+	suite.Run(t, new(ProjectPutRotateSuite))
+}
+
+func (s *ProjectPutRotateSuite) SetupTest() {
+	s.NoError(db.ClearCollections(serviceModel.RepoRefCollection, user.Collection))
```

Comment:
I'm assuming this is just copy pasta but you shouldn't need to clear any collections since you don't insert anything

Reply:
done